server/src/test/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImplTest.java

```diff
@@ -278,7 +282,6 @@ public void setUp() throws Exception {
                 nullable(String.class), nullable(Hypervisor.HypervisorType.class), nullable(Map.class), nullable(VirtualMachine.PowerState.class))).thenReturn(userVm);
         NetworkVO networkVO = Mockito.mock(NetworkVO.class);
         when(networkVO.getGuestType()).thenReturn(Network.GuestType.L2);
-        when(networkVO.getBroadcastUri()).thenReturn(URI.create(String.format("vlan://%d", instanceNic.getVlan())));
```

Comment:
why? Not used anymore/

Reply:
This mocking was not needed (tests pass after removal) after refactoring in other PRs (not this PR). So after rebasing, I was getting Mockito failures and so is not needed anymore.